src/ui/public/courier/fetch/request/__tests__/segmentedCreateQueue.js

```diff
@@ -0,0 +1,62 @@
+describe('ui/courier/fetch/request/segmented/_createQueue', () => {
```

Comment:
Also, by testing _createQueue directly rather than just its side effects on the SegmentedReq constructor, we're kind of breaking the `private` nature of the underscore syntax for functions like this.  I don't have strong opinions about it, but if you don't want plugin authors and such to start invoking underscore functions directly, we probably shouldn't be defining behaviors for them through specs.